backend/Services/CoordinateFetcher.php

```diff
@@ -49,8 +50,8 @@ public function coordinatesFromAddress(string &$currentCoordinates, string $addr
     public function fetchAddressToString(FetchAddress $addressObject)
     {
         return $addressObject->getStreet()
-            . $addressObject->getCity()
-            . $addressObject->getCountry()
-            . $addressObject->getZip();
+            . "," . $addressObject->getCity()
+            . "," . $addressObject->getCountry()
+            . "," . $addressObject->getZip();
```

Comment:
Maybe use `implode()` function.
https://www.php.net/manual/en/function.implode.php

Also add return type to the method.